templates/mentorship_relation_accepted.html

```diff
@@ -0,0 +1,11 @@
+<p>Hi {{ sender }},</p>
+<br>
+<p>Congratulations!!</p>
```

Comment:
double exclamations ( `!!` ) are not good to use in formal (or semi-formal) emails.